UI.AutomationTests/Admin/Booking/EditMultiDayHearingTests.cs

```diff
@@ -131,11 +131,14 @@ private SummaryPage EditMultiDayHearing(BookingDto hearingDto, SummaryPage summa
             
             // Go to the summary page
             // The journey is slightly different here when special measures is enabled
-            var specialMeasuresEnabled = FeatureToggle.Instance().SpecialMeasuresEnabled();
-            if (specialMeasuresEnabled)
-                summaryPage = videoAccessPointsPage.GoToSummaryPage();
-            else
-                summaryPage = videoAccessPointsPage.GoToOtherInformationPage().GoToSummaryPage();
+            // TODO: confirm steps with a dev
+            summaryPage = videoAccessPointsPage.GoToSummaryPage();
+            
+            // var specialMeasuresEnabled = FeatureToggle.Instance().SpecialMeasuresEnabled();
+            // if (specialMeasuresEnabled)
+            //     summaryPage = videoAccessPointsPage.GoToSummaryPage();
+            // else
+            //     summaryPage = videoAccessPointsPage.GoToOtherInformationPage().GoToSummaryPage();
```

Comment:
This linked to https://tools.hmcts.net/jira/browse/VIH-11118 ?

Reply:
reverting